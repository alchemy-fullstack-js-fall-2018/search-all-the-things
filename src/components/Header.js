import React, { Fragment } from 'react';

const Header = () => {

  return (
    <Fragment>
      <h1>Headlines Now!</h1>
      <h2><i>Your #1 Source for the News</i></h2>
    </Fragment>
  );
};

export default Header;
