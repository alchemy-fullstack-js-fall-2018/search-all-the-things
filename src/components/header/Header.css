.header {
  background-color: black;
  color: silver;
}
