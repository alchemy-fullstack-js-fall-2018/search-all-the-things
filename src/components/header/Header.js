import React, { Component } from 'react';

import styles from './Header.css';

export default class Header extends Component {


  render() {

    return (
      <header className={styles.header}>
        <h1>Hot Off the Motherfucking Presses</h1>
        <p>I mean that</p>
      </header>
    );
  }


}
