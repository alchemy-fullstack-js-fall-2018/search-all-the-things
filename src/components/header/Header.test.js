import Header from './Header';

describe('Header Component', () => {
  it('should be a header', () => {
    const header = new Header();
    expect(header.render()).toMatchSnapshot();
  });
});
