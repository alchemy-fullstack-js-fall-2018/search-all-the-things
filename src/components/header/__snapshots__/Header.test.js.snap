// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component should be a header 1`] = `
<header
  className="header"
>
  <h1>
    Hot Off the Motherfucking Presses
  </h1>
  <p>
    I mean that
  </p>
</header>
`;
